_includes/footer.html

```diff
@@ -34,6 +34,7 @@ <h6>Knowledge Base</h6>
                         <li><a href="{{ site.baseurl }}/docs/">Documentation</a></li>
                         <li><a href="https://omero-guides.readthedocs.io/">User Guides</a></li>
                         <li><a href="https://www.openmicroscopy.org/forums">Forum</a></li>
+                        <li><a href="https://www.openmicroscopy.org/security">Security Advisories</a></li>
```

Comment:
Should this link to https://www.openmicroscopy.org/security/advisories/ directly then?

Reply:
Yeah, I wasn't sure either. There's not a link from the list (which is certainly more of a "knowledge base") back to the top, so this seemed safer.

Reply:
Agreed. Interesting individual advisories have a back link to the list of advisories - https://www.openmicroscopy.org/security/advisories/2021-SV4/. 

So it feels like having a `back to Security` in https://www.openmicroscopy.org/security/advisories/ would be valuable either way. It does not need to happen right in this PR though.